content/en/blog/_posts/2019-09-02-kubernetes-with-microk8s.md

```diff
@@ -0,0 +1,149 @@
+---                                           
+title: 'Running Kubernetes locally on Linux with Microk8s'                                                           
+date: 2019-09-02                                 
+---
+**Authors**: [Ihor Dvoretskyi](https://twitter.com/idvoretskyi), Developer Advocate, Cloud Native Computing Foundation; [Carmine Rimi](https://twitter.com/carminerimi), Product Manager, Canonical
+
+This is article #2 in a series about local deployment options on Linux, and it will cover [MicroK8s](https://microk8s.io/)  - the click-and-run solution for deploying a Kubernetes cluster locally, originally developed by Canonical, the publisher of Ubuntu.
```

Comment:
It might be worth linking to article #1 if folks want to catch up before starting this one.

Reply:
Definitely throw that link in there.